cmd/substreams/init.go

```diff
@@ -401,3 +411,35 @@ func GetContractABI(ctx context.Context, contract eth.Address) (string, *eth.ABI
 
 	return abiContent, ethABI, nil
 }
+
+func GetContractCreationBlock(ctx context.Context, contract eth.Address) (string, error) {
+	req, err := http.NewRequestWithContext(ctx, "GET", fmt.Sprintf("https://api.etherscan.io/api?module=account&action=txlist&address=%s&page=1&offset=1&sort=asc&apikey=YourApiKeyToken", contract.Pretty()), nil)
```

Comment:
@YaroShkvorets since `apikey` isn't being replaced, could be remove, the 5 call/second would still be applied without providing API key.

Also, would it be possible to include the other Etherscan networks.

- https://api-goerli.etherscan.io/api
- https://api-sepolia.etherscan.io/api

There's some more as well such as BNB & Polygon:
- https://api.bscscan.com/api
- https://api.polygonscan.com/api

Reply:
Yeah currently generator works for Ethereum mainnet.

<img width="440" alt="image" src="https://github.com/streamingfast/substreams/assets/29608734/63b32b62-71f2-489a-8cc8-5c3aa00ec4e2">

Ideally the wizard should ask for the network (Mainnet, BNB, Polygon) after protocol. But this is a separate issue.